sockpuppet/routing.py

```diff
@@ -1,12 +1,17 @@
 from django.urls import re_path
+try:
+    from django.core.asgi import get_asgi_application
+except ImportError:
+    from channels.routing import get_default_application as get_asgi_application
 from channels.auth import AuthMiddlewareStack
 from channels.routing import ProtocolTypeRouter, URLRouter
 from sockpuppet.consumer import SockpuppetConsumer
 
-socket_patterns = [re_path(r'ws/sockpuppet-sync', SockpuppetConsumer)]
+socket_patterns = [re_path(r'ws/sockpuppet-sync', SockpuppetConsumer.as_asgi())]
```

Comment:
You might have already seen this -> https://channels.readthedocs.io/en/stable/releases/3.0.0.html#update-to-asgi-3 But in case you haven't. 

This means that if we're using channels <2.4 `as_asgi` doesn't exist and won't work. 

In the release notes it mentions that __call__ is used instead __init__ which is no longer used. So that needs to be fixed as well.